esphome/dth22.yaml

```
@@ -0,0 +1,36 @@
+---
+# vindriktning 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)